app/code/Magento/Quote/Model/ShippingAddressManagement.php

```diff
@@ -137,4 +139,13 @@ public function get($cartId)
         /** @var \Magento\Quote\Model\Quote\Address $address */
         return $quote->getShippingAddress();
     }
+
+    private function validateBeforeMerge($quote, $address)
```

Comment:
Please, add strict types to the method signature and generate docblock.

Reply:
Fixed